u6-spas/vscode/setting.json

```diff
@@ -0,0 +1,51 @@
+{
+    
+    "editor.codeActionsOnSave": [
+      "source.addMissingImports", 
+      "source.formatDocument", 
+      "source.fixAll.eslint", 
+      "source.fixAll.stylelint" 
+    ],
+    "editor.formatOnSave": false, 
+    "editor.formatOnPaste": true,
+    "editor.quickSuggestions": { "strings": true },
+    "editor.rulers": [80], 
+    "editor.tabSize": 2,
+    "eslint.format.enable": false,),
```

Comment:
ต่อท้าย `),` ไม่ควรอยู่ตรงนั้น ที่จริงฉันคิดว่าทั้งบรรทัดนี้สามารถลบออกได้ในขณะนี้ (เลิกใช้แล้ว)
The trailing `),` shouldn't be there. Actually I think this entire line can be removed now (deprecated).